recipe/meta.yaml

```
@@ -1,5 +1,6 @@
 {% set name = "nslsii" %}
 {% set version = "0.11.2" %}
+{% set python_min = "3.11" %}
```

Can we do 3.10? We still deploy 3.10 envs on the floor.
```suggestion
{% set python_min = "3.10" %}
```